docs/release/docker_image_support_matrix.rst

```diff
@@ -11,6 +11,25 @@ containers on dockerhub. The following tags, and associated inventories, are val
 
         .. tab-set::
 
+            .. tab-item:: Ubuntu 22.04
+
+                Tag: `rocm/pytorch:rocm5.7_ubuntu22.04_py3.10_pytorch_2.0.1 <https://hub.docker.com/layers/rocm/pytorch/rocm5.7_ubuntu22.04_py3.10_pytorch_2.0.1/images/sha256-21df283b1712f3d73884b9bc4733919374344ceacb694e8fbc2c50bdd3e767ee>`_
```

Comment:
5.7 or 5.7.1? @saadrahim

Reply:
5.7, this is not related to the 5.7.1 release.